pkg/executor/test/jointest/hashjoin/hash_join_test.go

```diff
@@ -258,9 +253,9 @@ func TestInlineProjection4HashJoinIssue15316(t *testing.T) {
 		"0 0 2",
 	))
 	// NOTE: the HashLeftJoin should be kept
-	tk.MustQuery("explain format = 'brief' select T.a,T.a,T.c from S join T on T.a = S.a where S.b<T.b order by T.a,T.c;").Check(testkit.Rows(
-		"Sort 12487.50 root  test.t.a, test.t.c",
-		"└─Projection 12487.50 root  test.t.a, test.t.a, test.t.c",
+	tk.MustQuery("explain format = 'brief' select /*+ HASH_JOIN_BUILD(T) */ T.a,T.a,T.c from S join T on T.a = S.a where S.b<T.b order by T.a,T.c;").Check(testkit.Rows(
+		"Projection 12487.50 root  test.t.a, test.t.a, test.t.c",
+		"└─Sort 12487.50 root  test.t.a, test.t.c",
```

Comment:
I am not sure why it was reordered. But it seems the new one is correct. Also, it doesn't change the query result.